lib/op-stream.js

```diff
@@ -24,33 +19,14 @@ inherits(OpStream, Readable);
 // themselves.
 OpStream.prototype._read = util.doNothing;
 
-OpStream.prototype.initProjection = function(backend, agent, projection) {
-  this.backend = backend;
-  this.agent = agent;
-  this.projection = projection;
-};
-
-OpStream.prototype.pushOp = function(collection, id, op) {
-  if (this.backend) {
-    var stream = this;
-    this.backend._sanitizeOp(this.agent, this.projection, collection, id, op, function(err) {
-      if (!stream.open) return;
-      stream.push(err ? {error: err} : op);
-    });
-  } else {
-    // Ignore any messages after unsubscribe
-    if (!this.open) return;
-    this.push(op);
-  }
-};
-
-OpStream.prototype.pushOps = function(collection, id, ops) {
-  for (var i = 0; i < ops.length; i++) {
-    this.pushOp(collection, id, ops[i]);
-  }
+OpStream.prototype.pushData = function(data) {
+  // Ignore any messages after unsubscribe
+  if (!this.open) return;
+  this.push(data);
```

Comment:
```suggestion
  // This data gets consumed in Agent#_subscribeToStream
  this.push(data);
```

^ To make it a bit easier to follow the flow of execution